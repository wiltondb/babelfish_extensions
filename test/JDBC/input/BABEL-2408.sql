EXEC sp_describe_undeclared_parameters 'COMMIT TRANSACTION;'
GO