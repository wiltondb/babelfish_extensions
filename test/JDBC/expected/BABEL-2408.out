EXEC sp_describe_undeclared_parameters 'COMMIT TRANSACTION;'
GO
~~START~~
int#!#varchar#!#int#!#nvarchar#!#smallint#!#tinyint#!#tinyint#!#int#!#varchar#!#varchar#!#varchar#!#nvarchar#!#int#!#varchar#!#varchar#!#varchar#!#bit#!#bit#!#bit#!#bit#!#bit#!#varchar#!#int#!#int
~~ERROR (Code: 11506)~~

~~ERROR (Message: An error occurred during the current command (Done status 0). Unsupported statement type for sp_describe_undeclared_parameters)~~

